### ISSUE TITLE
IPTorrent returning extra forward slash in url

### ISSUE THREAD

**Issue by lox** [Jackett/Jackett#143]
I'm seeing urls generated in the feed that looks like:

```
https://iptorrents.com//download.php/1648210/Transparent.S02E03.720p.WEBRip.x264-2HD.torrent
```

Note the double forward slash. It results in a 404. The correct URL is the one with a single double slash, which is returned by IPTorrent's search feature.

**Comment by flightlevel** [comment]
What version of Jackett?

**Comment by JigSawFr** [comment]
@lox Fixed in version 0.7.75 (https://github.com/Jackett/Jackett/commit/95e3615bfa461366b386122232d3fe91bb3f382c)

**Comment by lox** [comment]
Can confirm that 0.7.75 fixes the issue, thanks!
